synfig-studio/src/gui/widgets/widget_defaults.cpp

```diff
@@ -196,8 +196,8 @@ Widget_Defaults::Widget_Defaults():
 		_widget_otln_color->set_tooltip_text(_("Outline Color"));
 
 		// fixed outline color widget size
-		Gtk::Alignment *widget_otln_color(manage(new Gtk::Alignment(Gtk::ALIGN_END, Gtk::ALIGN_END, 0.0, 0.0)));
-		widget_otln_color->add(*_widget_otln_color);
+		_widget_otln_color->set_halign(Gtk::ALIGN_END);
```

Comment:
For consistency, please rename it without the prepended `_`. :)
And for better reading, without guessing, `outline` and not `otln` :P

Reply:
And please rephrase your commit ;)

Reply:
Renamed the widget_otln to outline ! And also rephrased the latest commit and headline of the issue ! Now resolving other widgets as well ! [^_^]

Reply:
I meant  "removing alignment test" ;) -> "remove deprecated Gtk::Alignment" would be better

Reply:
I guess I can't change name of the previous commit that I've already pushed ! I've tried renaming them and pushing them again but github won't show any change ! So, I will keep that in mind from next time [^_^]